apps/pie-docs/src/assets/styles/components/_contentLayout.scss

```diff
@@ -13,18 +13,21 @@
       flex: 0 1 calc(50% - (var(--dt-spacing-e) / 2));
     }
   }
+}
 
+.c-columns--three {
+  @include f.media('>=huge') {
+    flex-flow: row wrap;
 
-  &.c-columns--three {
-    @include f.media('>=huge') {
-      flex-flow: row wrap;
-
-      > * {
-        // stylelint-disable-next-line declaration-block-no-redundant-longhand-properties
-        flex: 0 1 calc(33.333% - var(--dt-spacing-e)); // Three equal-width columns
-      }
+    > * {
+      // stylelint-disable-next-line declaration-block-no-redundant-longhand-properties
+      flex: 0 1 calc(33.333% - var(--dt-spacing-e)); // Three equal-width columns
     }
   }
+
+  .c-contentPage-img {
+    height: 100%;
```

Comment:
this change is needed to ensure that ll images in the three-column layout have the same height.